src/continuous/continuous-observer.test.ts

```
@@ -198,12 +206,654 @@ describe('ContinuousObserver Integration', function () {
       newScheduler.restoreFromState(loadedState!);
 
       // Should have 2 remaining observations for first gateway
-      const remainingCount =
-        newScheduler.getSchedule().get(firstGateway)?.length ?? 0;
+      const remainingCount = newScheduler
+        .getSchedule()
+        .filter((scheduledObservation) => scheduledObservation.fqdn === firstGateway).length;
```

ESLint

[error] Replace `(scheduledObservation)·=>·scheduledObservation.fqdn·===·firstGateway` with `⏎··········(scheduledObservation)·=>·scheduledObservation.fqdn·===·firstGateway,⏎········`

(prettier/prettier)